angular.json

```diff
@@ -28,7 +28,10 @@
               "src/assets"
             ],
             "styles": [
+              "./node_modules/bootstrap/dist/css/bootstrap.min.css",
```

Comment:
we already use overridden bootstrap from stylesheet.css. Do you need to reference it here as well?